awx/ui_next/src/screens/Template/Survey/SurveyListItem.jsx

```diff
@@ -119,7 +119,11 @@ function SurveyListItem({
               )}
               {[question.type].includes('multiselect') &&
                 question.default.length > 0 && (
-                  <ChipGroup numChips={5}>
+                  <ChipGroup
+                    numChips={5}
+                    expandedText={i18n._(t`Show Less`)}
+                    collapsedText={i18n._(t`more`)}
```

Comment:
This seems inconsistent in capitalization etc, can you run it by @tvo318 ?

Reply:
This is the default value that comes from here.
https://www.patternfly.org/v4/documentation/react/components/chipgroup

Reply:
oh that makes sense! sounds good